assets/build/blocks/block.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/block.json",
+  "apiVersion": 2,
+  "name": "tsml/meetings",
+  "title": "Meetings",
+  "description": "12 Step Meeting List blocks",
+  "category": "embed",
+  "icon": "groups",
+  "attributes": {
+    "blockType": {
+      "type": "string",
+      "default": "tsml_ui"
+    },
+    "count": {
+      "type": "string",
+      "default": "5"
+    },
+    "message": {
+      "type": "string",
+      "default": "There are no upcoming meetings today. Please check back tomorrow."
```

Comment:
is this text accurate? this seems like it applies more to the Upcoming Meetings widget?

if this block embeds TSML UI it should probably say something like "TSML UI could not be embedded" - not sure what scenario this wouldn't show…

Reply:
This is a config value the user can change in the block settings when there are no upcoming meetings listed. It only shows up in the block config for that block, but needs a starting value.

Reply:
This is where that setting shows up.

<img width="277" alt="Screenshot 2024-03-23 at 10 04 21 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/9162021/1a97734d-e882-4c09-a5fe-0c0ce5972f46">